src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -165,32 +166,33 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
   @Test
   fun `complete draft adjudication`() {
     prisonApiMockServer.stubPostAdjudication()
-    bankHolidayApiMockServer.stubGetBankHolidays()
+    val intTestData = IntTestData(webTestClient, jwtAuthHelper, bankHolidayApiMockServer, prisonApiMockServer)
 
-    val draftAdjudicationResponse = dataAPiHelpers().startNewAdjudication(dateTimeOfIncident = DATE_TIME_OF_INCIDENT)
-    dataAPiHelpers().addIncidentStatement(draftAdjudicationResponse.draftAdjudication.id, "test statement")
```

Comment:
I am slowly removing all refs to dataAPiHelpers() in place of IntTestData, as the latter is more coherent

Reply:
Cool